modifiers/ops_abstractor.h

```diff
@@ -24,7 +24,9 @@ namespace pono {
 class OpsAbstractor : public Abstractor
 {
  public:
-  OpsAbstractor(const TransitionSystem & conc_ts, TransitionSystem & abs_ts);
+  OpsAbstractor(const TransitionSystem & conc_ts,
+                TransitionSystem & abs_ts,
+                bool abs_using_free_symbols);
```

Comment:
```suggestion
                bool abs_using_free_symbols = false);
```
This is a public API, so we should provide a default argument here to not break backwards-compatibility.